src/main/java/com/recap/updater/deletions/DeleteInfoProcessor.java

```diff
@@ -258,7 +258,7 @@ private String transformToDeleted(Map<String, Object> record, String bibOrItem,
         logger.info("deleting the individual item: " + owningInstitution + "-"
             + record.get(OWNING_INSTITUTION_ITEM_ID));
         deletedRecord.put("id", record.get(OWNING_INSTITUTION_ITEM_ID));
-        deletedRecord.put(BIB_IDS, record.get(BIB_IDS));
+        deletedRecord.put(BIB_IDS, Arrays.asList());
```

Comment:
Small question/suggestion would be, if we are setting bibIds to Arrays.asList() (an empty list) in the functions above, why do we have to set it again in transformToDeleted again? TransformToDeleted function can be agnostic about what the value for bibIds is supposed to be and just do `record.get(BIB_IDS)`, right? Thoughts?

Reply:
Sold! We centralized the logic in 49e0c80dc4f5265f62011c2a6cc36cc4827e1455